src/main/java/org/folio/spring/scope/FolioExecutionScopeExecutionContextManager.java

```diff
@@ -82,7 +82,7 @@ static String getConversationIdForScope() {
   static Map<String, Object> getFolioExecutionScope() {
     Map<String, Object> folioExecutionScope = folioExecutionScopeHolder.get();
     if (folioExecutionScope == null) {
-      log.warn("FolioExecutionScope is not set up. Fallback to default FolioExecutionScope.");
+      log.debug("FolioExecutionScope is not set up. Fallback to default FolioExecutionScope.");
```

Comment:
Let's keep it log.warn, because this is an unexpected case, and it means that something is wrong with the application logic.

Reply:
Done